config/locales/forms.yml

```diff
@@ -2,7 +2,7 @@ en:
   buttons:
     accept_all_cookies: Accept all cookies
     accept_and_continue: Accept and continue
-    add_another_break: add a reason for this break
```

Comment:
this variable name made me think that this link was for adding an additional break in work history, rather than adding a reason for the break in work history. I changed the name to better reflect what it was for (imo).